ethtools/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+description = "Ethcore Ethereum tools"
+homepage = "http://ethcore.io"
+license = "GPL-3.0"
+name = "ethtools"
```

Comment:
I just thought that any import/export operation or anything added in future that is not part of our client regular work should go here
So that it will be easy to delete this code or add new and not break other parts